src/gui/ViewDialog.cpp

```diff
@@ -345,16 +345,17 @@ void ViewDialog::createDialogContent()
 	connect(ui->pushButtonAddRemoveLandscapes, SIGNAL(clicked()), this, SLOT(showAddRemoveLandscapesDialog()));
 
 	// Grid and lines
-	connectGroupBox(ui->celestialSphereGroupBox,				"actionShow_Gridlines");
+	connectGroupBox(ui->celestialSphereGroupBox,			"actionShow_Gridlines");
```

Comment:
again, for readability, spaces, not tabs, should be used to align the second column.